proto/rig/api/v1/project/service.proto

```diff
@@ -104,10 +114,12 @@ message GetLicenseInfoResponse {
 
 message GetCustomObjectMetricsRequest {
   api.v1.capsule.ObjectReference object_reference = 1;
+  string project_id = 2;
```

Comment:
Should have environment as well. The objects are scoped to a specific kubernetes namespace